packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -163,6 +164,10 @@ export class FluidDataStoreRuntime
 		return this.dataStoreContext.IFluidHandleContext;
 	}
 
+	public get idCompressor(): IIdCompressor | undefined {
+		return this.dataStoreContext.idCompressor;
```

Comment:
Note that dataStoreRuntime is in a different compatibility layer than dataStoreContext / containerRuntime. And we support N/N-1 compat across this layer. This means if this PR goes into internal.5.x, idCompressor will be fully available in dataStoreRuntime in internal.6.x.

Reply:
Is there anything to be done specifically with regard to this compat layer? I couldn't think of anything that needed to be done besides the compressor possibly being undefined.